renderdoccmd/android/Loader.java

```diff
@@ -17,6 +18,11 @@ protected void onCreate(android.os.Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         getWindow().addFlags(WindowManager.LayoutParams.FLAG_KEEP_SCREEN_ON);
 
+        Context context = getApplicationContext();
+        if(context != null) {
+            context.startService(new Intent(this, DummyService.class));
```

Comment:
Should we also stop the foreground service when the activity is destroyed, or if the remote server exits for whatever reason?